src/main/java/io/reactivex/observers/LambdaConsumerIntrospection.java

```diff
@@ -0,0 +1,23 @@
+package io.reactivex.observers;
+
+import io.reactivex.annotations.Experimental;
+
+/**
+ * An interface that indicates that the implementing type is composed of individual components and exposes information
+ * about their behavior.
+ *
+ * <p><em>NOTE:</em> This is considered a read-only public API and is not intended to be implemented externally.
+ *
+ * @since 2.1.4 - experimental
+ */
+@Experimental
+public interface LambdaConsumerIntrospection {
+
+    /**
+     * @return {@code true} if a custom onError consumer implementation was supplied. Returns {@code false} if the
+     * implementation is missing an error consumer and thus using a throwing default implementation.
+     */
+    @Experimental
+    boolean hasCustomOnError();
```

Comment:
Shouldn't we provide `hasDefaultOnError()` instead? cc @JakeWharton 

Because if you think about it from RxJava perspective — that's what library can say about itself: is something  has a *default* value/implementation in compare to what library thinks is default. 

`hasCustomOnError()` basically checks for `hasDefaultOnError()` and then negates the result which is kinda confusing. 

TL;TR: "custom" in the API feels wrong, but maybe it's just me :)